docs/reference/quickstart/aggs-tutorial.asciidoc

```diff
@@ -0,0 +1,530 @@
+[[aggregations-tutorial]]
+== Analyze ecommerce data with aggregations using Query DSL
+++++
+<titleabbrev>Basics: Analyze ecommerce data with aggregations</titleabbrev>
+++++
+
+This hands-on tutorial shows you how to analyze ecommerce data using {es} aggregations with the `_search` API and Query DSL.
+
+You'll learn how to:
+
+* Calculate key business metrics like average order value
+* Analyze sales patterns over time
+* Compare performance across product categories 
+* Track moving averages and cumulative totals
+
+[discrete]
+[[aggregations-tutorial-requirements]]
+=== Requirements
+
+You'll need:
```

Comment:
This implies that you need these set up the cluster first, but start local does that for you? Should start local come first, then loading the data into Kibana?

Reply:
Agree this section needs some re-org. It's not clear that start local is the preferred method for creating the ES cluster, and that it should be called before trying to load the data

Reply:
Roger that!